web/web.go

```diff
@@ -130,22 +137,27 @@ func getTemplateFile(name string) (string, error) {
 }
 
 func getConsoles() string {
-	if _, err := os.Stat(*consoleTemplatesPath + "/index.html"); !os.IsNotExist(err) {
-		return "/consoles/index.html"
+	if _, err := os.Stat(*urlPrefix + *consoleTemplatesPath + "/index.html"); !os.IsNotExist(err) {
+		return *urlPrefix + "consoles/index.html"
 	}
 	if *userAssetsPath != "" {
-		if _, err := os.Stat(*userAssetsPath + "/index.html"); !os.IsNotExist(err) {
-			return "/user/index.html"
+		if _, err := os.Stat(*urlPrefix + *userAssetsPath + "/index.html"); !os.IsNotExist(err) {
+			return *urlPrefix + "user/index.html"
 		}
 	}
 	return ""
 }
 
+func getUrlPrefix() string {
+	return *urlPrefix
+}
+
 func getTemplate(name string) (t *template.Template, err error) {
 	t = template.New("_base")
 	t.Funcs(template.FuncMap{
 		"since":       time.Since,
 		"getConsoles": getConsoles,
+		"getUrlPrefix": getUrlPrefix,
```

Comment:
In Go, getters are usually not called `getThing`, just `thing` (perhaps `getConsoles` should also be changed then, though you could argue that it's not just a getter, but actually does quite some stuff - up for review). Anyways, I'd just call this `urlPrefix` (you might need to call the Go function differently though since `urlPrefix` is already a variable).